addons/website_event_booth/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_event_booth_public,event.booth.public,event_booth.model_event_booth,,1,0,0,0
```

Comment:
Note that we generally call it "access_event_booth_all", as this is a "generic rule", not specific to public (will be the same for portal for example).